style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  font-family: Tahoma, Geneva, sans-serif;
+}
+
+.main {
+  background-image: url(https://dl.airtable.com/.attachmentThumbnails/ad97dd00af8cdd4a666f5cf5a0d56bcc/451c6cae);
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  overflow-y: scroll;
+  font-size: 24px;
+  color: #fff;
+}
+
+.main::after {
+  content: "";
+  position: relative;
+  color: black;
```

Comment:
use Flexbox and Grid for website layout

Reply:
Just as you have done in the previous comments, I suggest you phrase your comments as suggestions rather than instrucions as this is more friendly and polite😉. also, kindly state why you are requesting a change for same reasons I shared in the previous comments😉.